anaconda_updates/scripts/show_version.sh

```diff
@@ -74,6 +74,10 @@ do
          PACKAGES_URL="fedora34/Packages/a"
          shift
          ;;
+        -f35)
+         PACKAGES_URL="fedora_devel/Packages/a"
```

Comment:
Of note: Path to packages is different on new cobra02.

Reply:
This script shouldn't really reflect cobra02 because it's not publicly accessible.

Reply:
OK, but then what should it reflect? The server at the top is nonsensical, so the paths make zero sense anyway.